policy-controller/k8s/api/src/policy/target_ref.rs

```diff
@@ -242,9 +236,9 @@ mod tests {
     #[test]
     fn cluster_targets_namespace() {
         let t = ClusterTargetRef {
+            group: None,
```

Comment:
Just being more explicit here (and all the test changes below)? These would all be the same as keeping `..Default::default()`.

Reply:
We removed the `Default` impl for `ClusterTargetRef` because it doesn't really make sense since there are required fields. An empty `name`, for instance, doesn't actually make sense for the type.

And now that there's no `Default` impl we can't use `..Default::default()`, to fill in the remaining values, unfortunately.